src/components/cohortPages/CohortDetails.astro

```diff
@@ -47,27 +47,44 @@ const { format } = Astro.props;
       team={[
         {
           name: "Raj Maitra",
-          desc: "we will have rewrite it in 3rd person tho.. but write a description of yoyrself in whatever form you",
+          desc: "Pen-and-paper engineer turned keyboard warrior in multiple rabbit holes. Often known for making a mess. An actual rabbit.",
```

Comment:
I feel even the descriptions of people we should keep in one place as a constant file so that when we change in one place it updates everywhere